pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -247,15 +247,15 @@ def __init__(self, size_word, threshold, data_ts, base_cardinality=2, max_card_a
         self.cdf_mean = None
         self.cdf_std = None
 
-        # attribut après une mise à jour de l'arbre
+        # Attribute after an update of the tree
```

Comment:
(my mistake, not clear in french) Boolean value passing True after

Reply:
```suggestion
        # Boolean value passing True after an update of the tree
```